drake/matlab/systems/CPPLINT.cfg

```diff
@@ -0,0 +1,6 @@
+# TODO(#2274) Fix this.
+filter=-runtime/references
+
+# TODO(#2271) Fix this.
+# Once matlab code is evicted, it should be a simple as removing this line.
```

Comment:
Right.  I have a few cpplint tune-ups on deck behind this PR (and the issue numbers will track that reasonably well).